services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/team/MembershipInvitationManagerImplTest.java

```diff
@@ -441,6 +441,18 @@ public void testUpdateId() {
 		}
 		assertTrue(caughtException);
 
+		// Token with null expiresOn should fail
+		token.setExpiresOn(null);
+		caughtException = false;
+		try {
+			membershipInvitationManagerImpl.updateInviteeId(userId, MIS_ID, token);
+		} catch (IllegalArgumentException e) {
```

Comment:
an alternative is to just put "fail();" after line 448.  The test will only reach this line if it fails to throw an exception.